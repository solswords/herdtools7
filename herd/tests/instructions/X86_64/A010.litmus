X86_64 A010
{
  uint64_t x;
  0:RBX=0x202020202020202;
}
  P0            ;
  MOVQ %RBX,(x) ;
locations [x;]
forall [x] = 0x202020202020202
