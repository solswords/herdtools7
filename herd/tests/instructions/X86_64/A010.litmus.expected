Test A010 Required
States 1
[x]=144680345676153346;
Ok
Witnesses
Positive: 1 Negative: 0
Condition forall ([x]=144680345676153346)
Observation A010 Always 1 0
Hash=a4f2c7b46c9fe04acb695c7d4ca4630b

